### DISCUSSION TITLE
怎么pull request

### DISCUSSION THREAD

**Discussion by kuanghy123** [OpenHUTB/hutb#119] [Q&A]
怎么pull request

**Comment by ZLW523523** [comment]
添加你的修改后在你的Code中点击Contribute然后就能open pull request了

**Comment by xiaocuia** [comment]
在自己的分支下新建文件描写自己的修改，保存在自己的分支下，然后拉去请求，会根据你的修改拉去请求，然后进行审核

**Comment by XieTJ** [comment]
我第一次开 PR 时也踩了坑，提醒大家一定要先 Fork 仓库，再在自己的 Fork 里修改，最后从 Fork 分支向原仓库发起 PR
